lib/ioki/model/operator/reporting/report_structure/item_group.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Ioki
+  module Model
+    module Operator
+      module Reporting
+        module ReportStructure
+          class ItemGroup < Base
+            attribute :type,
+                      on:   :read,
+                      type: :string
+
+            attribute :name,
+                      on:   :read,
+                      type: :string
+
+            attribute :label,
+                      on:   :read,
+                      type: :string
+
+            attribute :description,
+                      on:   :read,
+                      type: :string
+
+            attribute :items,
+                      on:         :read,
+                      type:       :array,
+                      class_name: ['Item', 'ItemGroup', 'Divider']
```

Comment:
Question: I saw there is also a Label class, why is this missing here?

Reply:
Good catch! :+1: I haven't yet added it because I'm unsure whether we're going to need it. This part of the API is still experimental (and I'm not sure yet how to use the label :slightly_smiling_face:) - so maybe this will disappear again.

I'll keep it out for now - let's see if we need to add it later. Thanks for the find though! :muscle:

Reply:
Alright! Thanks for the explanation 👍